dev-tools/build-packages/deb/Docker/Dockerfile

```diff
@@ -1,16 +1,17 @@
 FROM debian:10
 
-ENV DEBIAN_FRONTEND noninteractive
+ENV DEBIAN_FRONTEND=noninteractive
 
 # Installing necessary packages
 RUN apt-get update && apt-get install -y --allow-change-held-packages apt-utils && \
     apt-get install -y --allow-change-held-packages \
     curl sudo wget expect gnupg build-essential \
-    devscripts equivs selinux-basics procps gawk
+    devscripts equivs selinux-basics procps gawk jq
 
 # Add the script to build the Debian package
-ADD builder.sh /usr/local/bin/builder
+ADD deb-builder.sh /usr/local/bin/builder
+ADD debian /usr/local/bin/debian
```

Comment:
**question**: what is this line copying? I do not see any file called `debian`. Could you clarify it?

Reply:
It's copying the debian [directory](https://github.com/wazuh/wazuh-dashboard/tree/4.10.2/dev-tools/build-packages/deb/debian), which contains the necessary files to build the package.